packages/contractkit/src/wrappers/Accounts.ts

```diff
@@ -198,13 +198,13 @@ export class AccountsWrapper extends BaseWrapper<Accounts> {
       )
       return toTransactionObject(
         this.kit,
-        this.contract.methods.authorizeValidatorSigner(
+        this.contract.methods.authorizeValidatorSignerWithPublicKey(
           signer,
-          pubKey,
           proofOfSigningKeyPossession.v,
           proofOfSigningKeyPossession.r,
-          // @ts-ignore Typescript does not support overloading.
-          proofOfSigningKeyPossession.s
+          proofOfSigningKeyPossession.s,
+          // @ts-ignore bytes type
```

Comment:
we can use something like:
```ts
type SolBytes = Array<string | number[]>
const toBytes = (input: any): SolBytes => input
const fromBytes = (input: SolBytes): any => input as any
```
from BaseWrapper to avoid the ts-ignore here